akka-cluster/src/main/mima-filters/2.5.x.backwards.excludes

```diff
@@ -1,2 +1,5 @@
 # #24710 remove internal ClusterReadView.refreshCurrentState
 ProblemFilters.exclude[DirectMissingMethodProblem]("akka.cluster.ClusterReadView.refreshCurrentState")
+
+# Disable remote watch and remote deployment outside Cluster #26176
+ProblemFilters.exclude[DirectMissingMethodProblem]("akka.cluster.ClusterRemoteWatcher.props")
```

Comment:
@raboof the `props` method signature was changed to 
`def props(failureDetector: FailureDetectorRegistry[Address], settings: RemoteSettings): Props

are you suggestion to make it `private[cluster]`? I'd think it might be a good idea.

Reply:
`ClusterRemoteWatcher` is already internal, I read Arnout's comment only as a confirmation that it is ok to add this filter. All good.